src/utils/misc.ts

```diff
@@ -51,3 +53,19 @@ export function errorFromHeaders(
 
   return undefined;
 }
+
+export function getSendHeaders(options?: SendOptions): Headers {
```

Comment:
rename getSendHeaders to addThreadContextHeaders

Reply:
Renamed